src/providers/UserProvider.tsx

```diff
@@ -40,8 +41,8 @@ export default function UserProvider({ children }: UserProviderProps): JSX.Eleme
 
       if (response.requestSucceeded) {
         setUser(response.user!);
-        if (response.user && !userState?.gtmInstrumented && (window as any).INIT_GTAG) {
-          setUserState({ gtmInstrumented: true });
+        if (response.user && !userAnalyticsState?.gtmInstrumented && (window as any).INIT_GTAG) {
+          store.dispatch(updateGtmInstrumented({ gtmInstrumented: true }));
```

Comment:
oops missed this, useAppDispatch here

Reply:
Oh, yes! I should be able to do it now with the <Provider> change. Thanks